PHP/PHP Source.sublime-syntax

```diff
@@ -9,7 +9,7 @@ variables:
   identifier: '\b{{identifier_start}}[[:alnum:]_]*\b'
   path: '\\?({{identifier}}\\)*{{identifier}}'
   regex_modifier: '[eimsuxADJSUX]*'
-  sql_indicator: \s*(?:SELECT|INSERT|UPDATE|DELETE|CREATE|REPLACE|ALTER)\b
+  sql_indicator: \s*(?:SELECT|INSERT|UPDATE|DELETE|CREATE|REPLACE|ALTER)(?=$|\s)
```

Comment:
I don't think we need to highlight `SELECT` as keyword in an EOF situation, if we don't do so in situations of it being the only keyword in a quoted string.

Reply:
@deathaxe that would fail one of added tests.

Reply:
`(?=\s)` would cover that use case as `\n` is part of `\s`. In eof the lookahead wouldn't match and thus `SELECT` would not be highlighted as keyword until a space or newline is added. With current solution SELECT is highlihted as keyword at the end of file.

Reply:
Okay. I see. `\s` auto covers `$` for non-EOF situation.